cmd/template/cluster/provider/capo.go

```diff
@@ -65,3 +70,96 @@ func WriteOpenStackTemplate(ctx context.Context, client k8sclient.Interface, out
 
 	return nil
 }
+
+func WriteOpenStackTemplateAppCR(ctx context.Context, config ClusterCRsConfig) error {
+	var userConfigConfigMapYaml []byte
+	var err error
+
+	clusterAppOutput := templateapp.AppCROutput{}
+	defaultAppsAppOutput := templateapp.AppCROutput{}
+
+	clusterAppConfig := templateapp.Config{
+		AppName:   config.Name,
+		Catalog:   "control-plane-catalog",
+		InCluster: true,
+		Name:      "cluster-openstack",
+		Namespace: fmt.Sprintf("org-%s", config.Organization),
+		Version:   config.ClusterAppVersion,
+	}
+
+	defaultAppsAppConfig := templateapp.Config{
+		AppName:   "openstack-default-apps",
+		Catalog:   "control-plane-catalog",
```

Comment:
```suggestion
		Catalog:   "default",
```

The app is only pushed to the default catalog right now. Since that's where the child apps are.

You can check this is in the Circle [config](https://github.com/giantswarm/default-apps-openstack/blob/3a994712a58b2183737201e85cbcbabee8498321/.circleci/config.yml#L12)